test/lib/eventFactory.test.js

```diff
@@ -156,27 +409,18 @@ describe('Event Factory', () => {
                     sha: 'ccc49349d3cffbd12ea9e3d41521480b4aa5de5f',
                     token: 'foo'
                 });
-                assert.strictEqual(pipelineMock.lastEventId, model.id);
+                assert.strictEqual(syncedPipelineMock.lastEventId, model.id);
                 Object.keys(expected).forEach((key) => {
                     if (key === 'workflow') {
                         assert.deepEqual(model[key], expected[key]);
+                    } else if (key === 'workflowGraph') {
+                        assert.deepEqual(model[key], expected[key]);
                     } else {
                         assert.strictEqual(model[key], expected[key]);
                     }
                 });
             })
         );
-
-        it('should only update lastEventId if type is pipeline', () => {
-            config.type = 'pr';
-
-            return factory.create(config).then((model) => {
-                assert.instanceOf(model, Event);
-                // lastEventId should not have been updated
-                assert.calledOnce(pipelineMock.update);
-                assert.strictEqual(pipelineMock.lastEventId, null);
```

Comment:
can you add this assertion to one of pr tests ?

Reply:
added!